modules/apps/object/object-test/src/testIntegration/resources/com/liferay/object/web/internal/portlet/action/test/dependencies/linked_object_folder.json

```diff
@@ -0,0 +1,221 @@
+{
+	"actions": {
```

Comment:
Do we really need those "actions"?

Reply:
See my previous comment, please

Reply:
We saw [this example](https://github.com/liferay/liferay-portal/blob/d59596a7c917810621592fe7523762b292c9de63/modules/apps/object/object-test/src/testIntegration/resources/com/liferay/object/web/internal/portlet/action/test/dependencies/account_entry_system_object_definition.json#L4-L9) and decided to leave the actions